tests/unit/lazyfree.tcl

```diff
@@ -174,4 +174,17 @@ start_server {tags {"lazyfree"}} {
         assert_equal [s lazyfreed_objects] 2
         $rd close
     }
+
+    test "Unblocks client blocked on lazyfree via REPLICAOF command" {
+        set rd [redis_deferring_client]
+
+        r debug populate 100000 key 1000
```

Comment:
btw, any reason to use `debug populate` rather than `populate`?
i usually prefer to avoid the first one unless i have to.
in which case the `needs:debug` can be dropped.